src/main/java/se/sundsvall/byggrintegrator/configuration/ExceptionAsXmlHandlerConfig.java

```diff
@@ -26,6 +26,11 @@
  */
 @Configuration
 public class ExceptionAsXmlHandlerConfig {
+
+	private ExceptionAsXmlHandlerConfig() {
```

Comment:
This doesn't work as this is a configuration class and needs a public constructor to work

Reply:
yup. This is should not be in here.